gists/frontend-pull-request-guidelines.md

```diff
@@ -108,8 +125,10 @@ This document details guidelines for reviewing frontend (especially LPL React-fl
 - Duplicating context or sowing confusion with a variable name (e.g., using a singular `participant` variable for an array of participants)
   - Refer to the [variable naming cheatsheet](https://github.com/kettanaito/naming-cheatsheet) for more examples
 - You probably don't need [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce). Most applications of reduce can be simplified and more performant when using [local mutation](https://typeofnan.dev/mutation-isnt-always-bad-in-javascript/)
+- Set optional booleans to default to `false` instead of `true`. Optional boolean props should need to be actively set, not enabled by default. If necessary, you can flop the logic (e.g., use `hide...` instead of `show...`, or `disable...` instead of `enable...`).
```

Comment:
Did you mean "flip"? I've rarely seen "flop" used without a preceding "flip" or "belly"